addon/components/mdc-tab-bar.js

```diff
@@ -35,11 +35,24 @@ export default Ember.Component.extend(MDCComponent, {
   layout,
   classNames: ['mdc-tab-bar'],
   classNameBindings: ['isIconsOnly:mdc-tab-bar--icon-tab-bar', 'isIconsWithText:mdc-tab-bar--icons-with-text', 'mdcClassNames'],
+  attributeBindings: ['style'],
   init() {
     this._super(...arguments);
     set(this, 'tabs', Ember.A([]));
     set(this, 'mdcIndicatorStyles', {});
   },
+  didInsertElement() {
+    this._super(...arguments);
+    if (get(this, 'register-tab-bar')) {
+      get(this, 'register-tab-bar')(this);
+    }
+  },
+  willDestroyElement() {
+    this._super(...arguments);
+    if (get(this, 'deregister-tab-bar')) {
```

Comment:
You won't have to wrap this or the other in an `if` if you add a default for `deregister-tab-bar` in the `Attributes` region.